docs/development/cloud.rst

```diff
@@ -0,0 +1,58 @@
+Developing in the cloud
+=======================
+
+The Warehouse development environment requires at least 4GB of RAM. Some users
+have struggled to make Warehouse performant during local development. This
+page provides instructions on how to develop Warehouse in the cloud to avoid
+resource constraints.
+
+Cloud server
+------------
+
+The first step is to get a Virtual Machine in the cloud. This could be an AWS
+EC2 instance, GCP Compute Engine, Digital Ocean droplet, etc. A machine with
+4GB RAM is sufficient if you only use it for Warehouse and nothing else.
+
+Note that there is a monthly cost associated with these instances. Depending
+on the cloud provider, instances are available for approximately $20 / month,
+and costs can be saved by terminating the instance if you don't plan to use it
+for a few days.
+
+Development setup
+-----------------
+
+SSH into the cloud server. Create a new SSH key on this server and associate
+it with your Github account. Clone the Warehouse repo and go through the
+normal setup instructions for building the Docker images and starting the
+containers.
```

Comment:
Maybe could be great have  a link to ```https://warehouse.pypa.io/development/getting-started/?highlight=Makefile#configure-the-development-environment```

Reply:
5eb398a updated, thanks for the suggestion

Reply:
Great